packages/dbf/dbf.0.2.0/opam

```diff
@@ -0,0 +1,38 @@
+opam-version: "2.0"
+synopsis: "DBF format parsing"
+description:
+  "This is a small library meant to parse dBase database files. Many cases/variants are not handled, but it might already be useful."
+maintainer: ["philippe.veber@gmail.com"]
+authors: ["Léo Andrès" "Philippe Veber"]
+license: "ISC"
+homepage: "https://github.com/pveber/dbf"
+bug-reports: "https://github.com/pveber/dbf/issues"
+depends: [
```

Comment:
It is one of our policy to list OCaml as the dependency. I'm guessing a lower bound of >= 4.02, but you are free to adjust it.
```suggestion
depends: [
  "ocaml" {>= "4.02"}
```